src/FocusAPI/Methods/User/AddUserFurniture.cs

```diff
@@ -19,11 +19,10 @@ public async Task<Unit> Handle(AddUserFurnitureCommand command, CancellationToke
             FocusAPI.Models.User user = _context.Users.First(u => u.Id == command.UserId);
```

Comment:
This line is an example of a Db call that could have async-await and a try catch block added to it. You could have a try-catch around these two calls and another around the save changes async call at the bottom. it's up to you, but it would help to have at least a try-catch that would catch and allow you to add explanations to any unexpected errors from the DB calls.

Reply:
Don't think you can await-async "First" calls with LINQ - are these the line you are referring to?  Working on the Try/Catches now, but not sure which lines you're targeting with the async-await!

Reply:
Yeah those were the lines I was referring to. Similarly to how you can't await ToList(), but you can await ToListAsync(), you can await FirstAsync() or FirstOrDefaultAsync(). [Example linked here](https://github.com/ChrispyPeaches/FocusFriends/blob/4daae811bb0da3fdba120e3a161b337c457b4573/src/FocusApp.Client/Methods/MindfulnessTip/GetMindfulnessTipByRatingLevel.cs#L73-L76)